maintainers/maintainer-list.nix

```diff
@@ -3514,6 +3514,12 @@
     githubId = 1608697;
     name = "Jens Binkert";
   };
+  jeremyschlatter = {
+    email = "github@jeremyschlatter.com";
+    github = "jeremyschlatter";
+    githubId = 5741620;
+    name = "Jeremy Schlatter";
+  };
```

Comment:
Please make this into a separate commit:

````
maintainers: add jeremyschlatter
```

Reply:
Done.